build.gradle

```diff
@@ -8,52 +8,35 @@ buildscript {
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:4.0.1'
-        classpath 'io.codearte.gradle.nexus:gradle-nexus-staging-plugin:0.21.2'
         classpath 'de.marcphilipp.gradle:nexus-publish-plugin:0.4.0'
+        classpath 'io.codearte.gradle.nexus:gradle-nexus-staging-plugin:0.21.2'
     }
 }
 
-allprojects {
-    repositories {
-        google()
-        jcenter()
-    }
+plugins {
+    id 'io.codearte.nexus-staging' version '0.21.2'
 }
```

Comment:
Nitpick - the other 2 gradle files have `plugins` at the top of the file. Should we do that here for consistency?

Reply:
Putting the plugins at the top in this file resulted in an error because gradle requires `buildscript {}` blocks to come before `plugins {}` blocks

Reply:
Good to know, thank you!